packages/stg-evm-v2/hardhat.config.ts

```diff
@@ -261,7 +261,7 @@ const networks: NetworksUserConfig = {
     },
     'klaytn-mainnet': {
         eid: EndpointId.KLAYTN_V2_MAINNET,
-        url: process.env.RPC_URL_KLAYTN_MAINNET || 'https://rpc.ankr.com/klaytn',
+        url: process.env.RPC_URL_KLAYTN_MAINNET || 'https://kaia.blockpi.network/v1/rpc/public',
```

Comment:
I changed these because they became unresponsive at one point, potentially because klaytn is being renamed to kaia